server_config_test.cc

```diff
@@ -20,130 +21,243 @@ class ServerConfigTest : public ::testing::Test {
   NginxConfig out_config_;
 };
 
+TEST_F(ServerConfigTest, BuildOK) {
+    std::unique_ptr<ServerConfig> server_config
+      = ParseString("port 8080; \
+                     path /echo EchoHandler {}");
+    bool ec = server_config->Build();
+
+    EXPECT_EQ(true, ec)
+      << "Build() should return true if a valid config is supplied to a \
+          Server Config is supplied";
```

Comment:
Minor copy-paste typo: "Build() should return true if a valid config is supplied to a Server Config"